app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -17,37 +17,79 @@ import androidx.compose.foundation.lazy.GridCells
 import androidx.compose.foundation.lazy.LazyVerticalGrid
 import androidx.compose.foundation.lazy.items
 import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Snackbar
+import androidx.compose.material.SnackbarDuration
+import androidx.compose.material.SnackbarHost
+import androidx.compose.material.SnackbarResult
 import androidx.compose.material.Surface
+import androidx.compose.material.rememberScaffoldState
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
 import org.mozilla.fenix.R
 import org.mozilla.fenix.theme.FirefoxTheme
 import org.mozilla.fenix.wallpapers.Wallpaper
 
 /**
- * The screen for controlling settings around Wallpapers.
+ * The screen for controlling settings around Wallpapers. When a new wallpaper is selected,
+ * a snackbar will be displayed.
  *
  * @param wallpapers Wallpapers to add to grid.
  * @param selectedWallpaper The currently selected wallpaper.
- * @param onSelectWallpaper Action to take when a new wallpaper is selected.
+ * @param onSelectWallpaper Callback for when a new wallpaper is selected.
+ * @param onViewWallpaper Callback for when the view action is clicked from snackbar.
  */
 @Composable
 fun WallpaperSettings(
     wallpapers: List<Wallpaper>,
     selectedWallpaper: Wallpaper,
     onSelectWallpaper: (Wallpaper) -> Unit,
+    onViewWallpaper: () -> Unit,
 ) {
-    Surface(color = FirefoxTheme.colors.layer2) {
-        WallpaperThumbnails(
-            wallpapers = wallpapers,
-            selectedWallpaper = selectedWallpaper,
-            onSelectWallpaper = onSelectWallpaper
-        )
+    val coroutineScope = rememberCoroutineScope()
+    val scaffoldState = rememberScaffoldState()
+    val snackbarMessage = stringResource(id = R.string.wallpaper_updated_snackbar_message)
+    val snackbarActionLabel = stringResource(id = R.string.wallpaper_updated_snackbar_action)
+    Scaffold(
+        backgroundColor = FirefoxTheme.colors.layer1,
+        scaffoldState = scaffoldState,
+        snackbarHost = { hostState ->
+            SnackbarHost(hostState = hostState) { data ->
+                Snackbar(
```

Comment:
**Blocking Change:** Looking at the screenshots, this doesn't seem to be styled correctly to our existing Fenix Snackbar (seems to be missing the font family at least). I am blocking on this to avoid shipping a Snackbar that doesn't conform to the existing design spec.

I think I also want to double check the colors used here in a future review.

Reply:
Updated the font family along with several other style alignments to the existing `fenix_snackbar` layout and [designs](https://www.figma.com/file/TebIgESfWmQkMcEmwNaYZl/Android?node-id=136%3A40).

Reply:
<img width="625" alt="image" src="https://user-images.githubusercontent.com/10427470/149568725-6324cb6e-43e9-45e4-bf01-af7c08243db1.png">